### DISCUSSION TITLE
Can't find a server

### DISCUSSION THREAD

**Discussion by taereea** [Tyrrrz/DiscordChatExporter#779] [Q&A] [answered]
I got my token and I can download chats from all my servers but one. How can I find it?

**Comment by 96-LB** [comment, marked as answer]
I'm not aware of anything that could cause this. Does the server happen to have no icon? In Discord, the default icon will look like some letters that spell an acronym of the server name, but since this isn't a real icon DiscordChatExporter is unable to load it. Instead, you'll see the default Discord avatar for that server's logo. If this is the case, you might just not be recognizing the server in your list.

**Reply by taereea** [reply]
Hello, the server does have an icon. However, I can't find it in the dropdown.

**Reply by 96-LB** [reply]
The servers are ordered by ID in the application. If you copy the ID of the server you're trying to find and compare it to others in the list, that could help you find where it should be.

**Reply by taereea** [reply]
Thank you, I was able to find it.
